src/test/java/com/wikia/webdriver/common/templates/TemplateNoFirstLoad.java

```diff
@@ -1,8 +1,30 @@
 package com.wikia.webdriver.common.templates;
 
+import com.wikia.webdriver.common.core.elemnt.JavascriptActions;
+import com.wikia.webdriver.common.core.url.Page;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.ad.VideoAd;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.ad.VuapVideos;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsBaseObject;
+
 public class TemplateNoFirstLoad extends NewTestTemplate {
 
   @Override
   protected void loadFirstPage() {
   }
+
+  protected AdsBaseObject openPageWithVideoInLocalStorage(Page page, VideoAd videoAd) {
+    final AdsBaseObject ads = new AdsBaseObject(driver, urlBuilder.getUrlForWiki("project43"));
+    putVASTToLocalStorage(videoAd.getVastXML());
+    ads.getUrl(page);
+    return ads;
+  }
+
+  protected AdsBaseObject openPageWithVideoInLocalStorage(Page page) {
+    return openPageWithVideoInLocalStorage(page, VuapVideos.DEFAULT);
+  }
+
+  private void putVASTToLocalStorage(String vast) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "putVASTToLocalStorage" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)